### PR TITLE
EAS-1865 Purging Test Alerts on Preview

### PR DESCRIPTION
**Pull request by jonathan-owens-gds**
- The conftest.py (test configuration) module calls the `/purge` endpoint in the API.
- The purge interval is configured in the /environment.sh file that sets up the environment variables.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,5 +25,6 @@ lint: clean
 test: clean ## Run functional tests against local environment
 	pytest -v -n auto --dist loadgroup \
 	tests/functional/preview_and_dev/test_broadcast_flow.py \
-	tests/functional/preview_and_dev/test_platform_admin_flow.py \
-	tests/functional/preview_and_dev/test_cbc_integration.py
+
+## tests/functional/preview_and_dev/test_platform_admin_flow.py \
```
Comment (jonathan-owens-gds) [inline Makefile:29]: These test will be reinstated in the next ticket.

File: environment.sh [hunk 2]
```diff
@@ -39,4 +39,6 @@ export PLATFORM_ADMIN_ID='c3d33860-a967-40cf-8eb4-ec1ee38a4df9'
 export PLATFORM_ADMIN_EMAIL='notify-tests-preview+local-broadcast3@digital.cabinet-office.gov.uk'
 export PLATFORM_ADMIN_NUMBER=07700900222
 
+export BROADCAST_SERVICE_NAME='Functional Tests Broadcast Service'
 export BROADCAST_SERVICE_ID='8e1d56fa-12a8-4d00-bed2-db47180bed0a'
+export FUNCTIONAL_TEST_ALERT_RETENTION_LIMIT=3
```
Comment (gds-gharper) [inline environment.sh:44]: I'd be tempted to indicate what this is, when reading this I was inclined to think this was the number of alerts beyond which we would delete.

Perhaps `FUNCTIONAL_TEST_PURGE_OLDER_THAN`

File: config.py [hunk 3]
```diff
@@ -119,6 +119,11 @@ def setup_preview_dev_config():
                     "mobile": os.environ["PLATFORM_ADMIN_NUMBER"],
                 },
                 "api_key_live": os.environ["BROADCAST_SERVICE_API_KEY"],
+                "service_name": os.environ["BROADCAST_SERVICE_NAME"],
+                "service_id": os.environ["BROADCAST_SERVICE_ID"],
+                "purge_retention_limit": os.environ[
```
Comment (gds-gharper) [inline config.py:124]: I'd be tempted to call this `purge_older_than`

File: tests/functional/preview_and_dev/test_broadcast_flow.py [hunk 4]
```diff
@@ -102,7 +102,7 @@ def test_prepare_broadcast_with_new_content(driver):
     assert current_alerts_page.is_text_present_on_page("since today at")
     alert_page_url = current_alerts_page.current_url
 
-    time.sleep(180)
+    time.sleep(10)
```
Comment (gds-gharper) [inline tests/functional/preview_and_dev/test_broadcast_flow.py:105]: I assume that these sleeps are there to wait until the site has built.  Can we not make a call and determine if the site is currently being built?  Probably out of scope for this change but might be worth creating a tech debt ticket for.

Comment (jonathan-owens-gds) [reply tests/functional/preview_and_dev/test_broadcast_flow.py:105]: If there was a "last modified" field on the index page, we could scrape that timestamp and watch for a change, but this is probably beyond the scope of this ticket.

File: environment.sh (hunk 2 above)
Comment (jonathan-owens-gds) [reply environment.sh:44]: done

File: config.py (hunk 3 above)
Comment (jonathan-owens-gds) [reply config.py:124]: done

**Review by gds-gharper — APPROVED** [review]

**Pull request merged**